changelog/19.0/19.0.0/release_notes.md

```diff
@@ -48,7 +48,7 @@ Vitess will however, continue to support importing from MySQL 5.7 into Vitess ev
 #### <a id="vttablet-flags"/>VTTablet Flags
 
 - The following flags — which were deprecated in Vitess 7.0 — have been removed:
-  `--vreplication_healthcheck_topology_refresh`, `--vreplication_healthcheck_retry_delay`, and `--vreplication_healthcheck_timeout`.
+`--vreplication_healthcheck_topology_refresh`, `--vreplication_healthcheck_retry_delay`, and `--vreplication_healthcheck_timeout`.
```

Comment:
Indentation level changed in error?

Reply:
From what I am seeing it doesn't affect the format on GitHub.